content/en/docs/tasks/traffic-management/egress/egress-tls-origination/mtls_test.sh

```diff
@@ -21,7 +21,6 @@ set -e
 set -u
 set -o pipefail
 
-source "content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/snips.sh"
```

Comment:
why is this not working anymore? is the exportTo part the only change?

Reply:
The issue is the the egress gateway mTLS tests used the same setup code but must not use export to. Removing this import breaks the dependency between the workload and gateway mTLS tests